com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/utils/MigrationCfgUtils.java

```diff
@@ -889,8 +889,17 @@ protected VerifyResultMessages createVerifyResult(
 
     /** @return true if user should change the target character type's size to avoid data lost. */
     public boolean doesNeedToChangeCharacterTypeSize() {
-        return DatabaseType.MYSQL.equals(config.getSourceDBType())
-                || DatabaseType.ORACLE.equals(config.getSourceDBType());
+        boolean isEffectedByCharacterTypeSize;
+
+        if (config.targetIsDBDump() || config.targetIsCSV()) {
```

Comment:
Changing it to `config.targetIsOnline()` is clearer.
If you use the code you wrote, you should also add SQL and XLS.

Reply:
change if condition statement. it cover all 'target file' migration